GOVERNANCE.md

```diff
@@ -0,0 +1,183 @@
+* Add new gpg keys to [our .github repository](https://github.com/pulumiverse/.github/blob/master/recipients.txt) and remove old keys
+* Ensure that everybody can [decrypt and encrypt](https://github.com/pulumiverse/.github#gpg) our secrets
+* Add the mail address from all new members to [our pmc@pulumiverse.com](https://app.mailgun.com/app/lists/pmc%40pulumiverse.com) mailing list and remove former members
+
+### Specialized Roles
+
+There are three officer roles in Pulumiverse. They can all be held by the same
+person (though this isn't ideal), as long as the work is getting done. They
+can have any other role in the organization, including being on the PMC. The
+PMC is responsible for requesting volunteers and appointing a volunteer to the
+role. Appointments last until either the officer steps down, or the unlikely
+and unfortunate situation where the PMC removes the officer. The PMC can add
+more officer roles as it sees fit by modifying this document.
+
+1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating
+
+2) Security Officer: This is the point of contact for external or internal security issues, this person has a published gpg key, and will be the main point of contact for CVE numbers and such
+
+3) Communications Officer: This is the main point of contact for external and internal publicity and marketing efforts and requests
+
+## Lazy Consensus
+
+Lazy consensus is a very important concept within the project. It is this
+process that allows a large group of people to efficiently reach
+consensus, as someone with no objections to a proposal need not spend time
+stating their position, and others need not spend time reading such statements.
+
+For lazy consensus to be effective, it is necessary to allow at least 72 hours
+before assuming that there are no objections to the proposal. This requirement
+ensures that everyone is given enough time to read, digest and respond to the
+proposal. This time period is chosen so as to be as inclusive as possible of
+all participants, regardless of their location and time commitments.
+
+We do require one affirmative vote as part of the Lazy consensus model.
+
+When the lazy consensus model does not converge on a decision or outcome, the
+PMC can step in and choose a path.
+
+## PMC Removal
+
+A member of the PMC can be removed before the end of their 1 year term by a
+supermajority (2/3) vote of the PMC. This can happen for a number of reasons,
+but the expected reason is following a Code of Conduct violation incident. Any
+report of problematic behaviour of PMC members will trigger an investigation
+and can trigger a vote. A report of the incident and actions taken (if any)
+will be published for the public. A member of the PMC can step down at any
+time.
+
+If a vacancy is left on the PMC, the PMC can optionally instruct the elections
+officer to perform another election or wait until the next election cycle.
```

Comment:
«Will trigger an investigation». By whom?
Who decides to trigger a vote? Anyone?

«by a supermajority (2/3) vote of the PMC». I don't think that is enough. Only PMCs are able to kick a PMC? The community needs to be able to. How?